core/generator.js

```diff
@@ -104,7 +105,11 @@ Blockly.Generator.prototype.workspaceToCode = function(workspace) {
       // Top-level blocks don't care about operator order.
       line = line[0];
     }
-    if (line) {
+
+    if(line == undefined){
+      blocksNotGenerated.push(block);
+    }
+    else if (line) {
```

Comment:
Google style: put else on the same line as the trailing curly brace.
Here and below.